GitUI/CommandsDialogs/FormArchive.cs

```diff
@@ -153,16 +153,16 @@ private string GetPathArgumentFromGui()
                 // 1. get all lines from text box which are not empty
                 // 2. wrap lines with ""
                 // 3. join together with space as separator
-                return string.Join(" ", textBoxPaths.Lines.Where(a => !a.IsNullOrEmpty()).Select(a => string.Format("\"{0}\"", a)));
+                return string.Join(" ", textBoxPaths.Lines.Select(a => a.QuoteNE()));
```

Comment:
Test QuoteNE()? What would that give?
I can revert the change and add the simplified code as a comment instead. (not doing that yet)

Reply:
It is not about testing `QuoteNE`.
The purpose of unit tests is to provide assurance that the new implementation does not regress. Are you 100% sure they are the same?

Reply:
Technically the following:
'''a

b
'''

becomes '"a"  "b"' (two spaces in between) instead of one, the data is used in a Git command so the space makes no difference.

This change is admittedly unrelated, and should have been included in some previous simplification PR. Since there are discussions about it, either the change was needed or we drop it.